src/styles/admin-core/content-blocks/_three-clickable-tiles.scss

```diff
@@ -0,0 +1,65 @@
+@use "src/styles/abstracts" as *;
+
+$component: "c-block-three-clickable-tiles";
+$tile-size: 27rem;
```

Comment:
pages can have different widths, so setting a hard width is not a very good solution:
![image](https://user-images.githubusercontent.com/1710840/232743069-33b3fcc4-62ed-4673-a718-79229b5ebd1a.png)
![image](https://user-images.githubusercontent.com/1710840/232743154-22351522-1096-47b0-88f1-52200529c1a9.png)

Its better to make each tile 33% wide and make the height the same as the width:
```css
.c-block-three-clickable-tiles__term {
    width: 33%;
}
.c-block-three-clickable-tiles__tile-image {
    display: flex;
    justify-content: center;
    align-content: center;
    width: 100%; // change this to 100%
    /* height: 27rem; */
    background-size: cover;
    background-position: center center;
    background-repeat: no-repeat;
    padding: 0 5rem;
    flex-wrap: wrap;
    text-align: center;
    aspect-ratio: 1 / 1; // Add aspect ratio
}
```
![image](https://user-images.githubusercontent.com/1710840/232743488-55e43452-338b-4f9e-a34e-1da4b0ff0be9.png)